### ISSUE TITLE
Webtorrent still makes itself the default app for .torrent files

### ISSUE THREAD

**Issue by goofrider** [webtorrent/webtorrent-desktop#1873]
System version: OSX Catalina 10.15.x
WebTorrent version: 0.23.0 (0.108.6)

**Comment by goofrider** [comment]
Just tried 0.24. It says it's the default app when it's not as I clicked on it one time before (to see if it'll allow me to unset it). It hasn't taken over as default tho just by running it.

Basically the same behavior as 0.23. And may potentially still mysteriously take over .torrent files some time later without asking.

Anyways it's just a minor annoyance, at least for those of us who know how to change the default app using other means like SwiftDefaultApps and Duti. I;m just submitting this in case others also have this issue,

**Comment by dsernst** [comment]
Hmm, this sounds frustrating. Thank you for filing this report.

**Comment by feross** [comment]
Yeah, this feature doesn't seem like it's working correctly anymore.

**Comment by idkwhojamesis** [comment]
Does it have to do with conflicting file handler ranks? Comparing `LSHandlerRank` for `.torrent` in `info.plist` in [Transmission](https://github.com/transmission/transmission/blob/3a96a5c316af70b16bae1ac487360c6af56a6b0d/macosx/Info.plist#L9-L26) vs. [Webtorrent](https://github.com/webtorrent/webtorrent-desktop/blob/8e86c855d3a9c782e72600c8f8f00c02e9fbb707/bin/package.js#L196-L204), they are both set to `Owner`, the highest priority. 

One interesting difference is that `CFBundleTypeRole` is set to `Viewer` in Transmission vs. `Editor` in Webtorrent; maybe this also makes a difference?

If [this older post about Mac file associations](https://stackoverflow.com/questions/26643721/overriding-mac-app-file-associations-via-cfbundledocumenttypes-in-info-plist) is still correct, the OS seems to set the earliest installed app as the default.
